aten/src/ATen/native/native_functions.yaml

```diff
@@ -2046,7 +2046,7 @@
     SparseCPU, SparseCUDA: expm1_sparse_out
     SparseCsrCPU, SparseCsrCUDA: expm1_sparse_csr_out
 
-- func: expand(Tensor(a) self, int[] size, *, bool implicit=False) -> Tensor(a)
+- func: expand(Tensor(a) self, BoxedInt[] size, *, bool implicit=False) -> Tensor(a)
```

Comment:
I think we could use a name closer to the fact that this is a size here.
Also why isn't this the same as `SymbolicOrConcreteInt` that the function takes?

Reply:
> Also why isn't this the same as SymbolicOrConcreteInt that the function takes?

I really don't have a good answer for this except for `SymbolicOrConcreteInt` seemed a bit too long to type.

> I think we could use a name closer to the fact that this is a size here

I'm not sure it's always true though. We may have an offset expression like t.size()[1] + 10 . Is it really a size?  @ezyang  was also wondering why it's not called SizeSomething but it seems it's a bit broader than just a size expression

Reply:
"DimSize"? 

And yeah, agree -- we should name `SymbolicOrConcreteInt` to whatever we decide here

Reply:
Or "AbstractInt", for some more bikeshedding

Reply:
I'm somewhat willing to generalize the name for a more general concept of symbolic ints, but it makes it more obscure when you should use it as opposed to "int". Need to think about it.